### ISSUE TITLE
Toggle camera reset

### ISSUE THREAD

**Issue by lchauvin** [Kitware/VolView#638]
Hello,

I saw that there is a variable to turn off the camera reset, but there is no UI.

I'm planning to add a toggle button to the settings to turn on / off the automatic camera reset.

Any objection or comments ?

**Comment by floryst** [comment]
Are you referring to `resizeToFit`? If so, sure. Having a setting to toggle its functionality is fine by me.

**Comment by lchauvin** [comment]
in `vtkSliceView`:

```
interface Props {
  viewId: string;
  imageId: Maybe<string>;
  viewDirection: LPSAxisDir;
  viewUp: LPSAxisDir;
  disableAutoResetCamera?: boolean;
}
```

There is a `disableAutoResetCamera` but there is not UI to toggle it.

**Comment by lchauvin** [comment]
What is the difference between `disabledAutoResetCamera` in `VtkSliceView`, `autoFit` in `useAutoFitState` and `noAutoReset` (that I don't really understand where it's defined / updated) in `VtkSliceView`:

```
watchImmediate([disableAutoResetCamera, viewID, imageID], ([noAutoReset]) => {
  if (noAutoReset) return;
  resetCamera();
});
```

**Comment by lchauvin** [comment]
Should I create a new `useAutoResetCameraState` file, or should I use the `resizeToFit` variable from `useAutoFitState` ?

**Comment by PaulHax** [comment]
Looks like the `useAutoFitState` is `VtkSliceView` internal detail.  `disabledAutoResetCamera` camera looks like a winner.

**Comment by floryst** [comment]
Indeed, you just want "disableAutoResetCamera" since noAutoReset is just the watched value of the former variable.

**Comment by lchauvin** [comment]
Is it a property that should be in CameraConfig (for each combination of view / data), or intrinsic to the camera properties ?

**Comment by floryst** [comment]
That's just a property exposed through the component. Right now we can just use a global toggle for that behavior, unless there are actual use-cases for having that toggle exist per-view/data combination.

**Comment by Jackie733** [comment]
So now in `vtkSliceView` file, the 'disableAutoResetCamera' property is useless or deprecated?

**Comment by lchauvin** [comment]
No, but it's coming from a pinia store now:

`const { disableCameraAutoReset } = storeToRefs(useViewCameraStore());`
